### DISCUSSION TITLE
[FEATURE] why celeborn not support flink 1.16

### DISCUSSION THREAD

**Discussion by y0908105023** [apache/celeborn#2523] [Q&A]
celeborn support 1.14/1.15/1.17/1.18/1.19, why is 1.16 not support.

**Comment by pan3793** [comment]
cc @RexXiong

**Comment by zhisheng17** [comment]
If we want to add support for Flink 1.16 to Celeborn's current codebase, what modifications need to be made? Why flink 1.16 version is not currently supported? @RexXiong

**Comment by mridulm** [comment]
We actually have prototyped 1.16 support internally, as we are still on Flink 1.16 currently ... my understanding is there were not enough users for that version and so was Celeborn did not support it (which is why we did not forward port it to master and contribute).

If that is not the case, and there is sufficient user requests for it, we can contribute the support.
+CC @venkata91

**Reply by RexXiong** [reply]
Thanks @mridulm,
It seems several community users have asked for Celeborn to support Flink 1.16. It would be greatly appreciated if you could contribute to providing this support. cc @zhisheng17

**Comment by mridulm** [comment]
Sounds good, let me work with @venkata91 on it @RexXiong; who will be doing most of the heavy lifting :-)

**Comment by zhisheng17** [comment]
@mridulm is there any issue i can follow?

**Comment by mridulm** [comment]
I created CELEBORN-1504
